harness/engine/network.go

```diff
@@ -0,0 +1,160 @@
+package engine
+
+import (
+	"fmt"
+	"github.com/dusk-network/dusk-blockchain/pkg/config"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"os"
+	"os/exec"
+	"time"
+)
+
+type Network struct {
+	Nodes     []*DuskNode
+	processes []*os.Process
+}
+
+// Bootstrap performs all actions needed to initialize and start a local network
+// This network is alive by the end of all tests execution
+func (n *Network) Bootstrap(workspace string) error {
+
+	initProfiles()
+
+	blockchainExec, blindBidExec, seederExec, err := n.getExec()
+	if err != nil {
+		return err
+	}
+
+	// Start voucher seeder
+	if len(seederExec) > 0 {
+		if err := n.start("", seederExec); err != nil {
+			return err
+		}
+	} else {
+		// If path not provided, then it's assumed that the seeder is already running
+		log.Warnf("Seeder path not provided.")
```

Comment:
Maybe it would be a good idea to mention what is said in the comment above, in the logging statement as well. (`it's assumed that the seeder is already running`)

Something like `Skipping startup of voucher seeder` or something could do.

Reply:
As a side note, I'm curious as to why you opted for assuming it is running, rather than failing if the path provided is not correct.

Reply:
I think it could be helpful to support the case where voucher is already started manually. That would allow us to run E2E testing against testnet itself.

E.g another test file could define this situation:

- configure N local nodes (incl blindibid)
- connect to testnet seeder
- perform blockchain assertions at the point of testnet tip (not from genesis height)

Reply:
Ah, I see. In that case, I'll just ask you to add a little bit of info to the log entry. It might be a bit cryptic if another developer is running into trouble with the tests and can't tell from terminal output that it's only the seeder that isn't online/defined.

Reply:
Noted.